geotrek/api/v2/filters.py

```diff
@@ -483,6 +484,16 @@ def get_schema_fields(self, view):
         )
 
 
+class TouristicEventFilterSet(filters.FilterSet):
+    bookable = filters.BooleanFilter(
+        field_name=_("Bookable"), 
```

Comment:
```suggestion
        field_name="bookable",
```
this is field in database